docs/projects/t1-2022/thoth-tech-leadership/weekly-announcements/week5.md

```diff
@@ -0,0 +1,71 @@
+# Week 5 General Announcement
+
+Good Evening Thoth Tech.
+
+Here is this week's update from your friendly neighbourhood leadership team.
+
+First and foremost, congratulations on the excellent feedback on Task 2.1! It
+was a team effort and everyone deserves a pat on the back.
+
+## OnTrack tasks
+
+For D/HD aiming students, you should start working on Task 5.1 retrospective.
```

Comment:
Task 5.1 isn't a retrospective - might get a bit confusing

Reply:
Addressed in [88e2bd0](https://github.com/thoth-tech/handbook/pull/48/commits/88e2bd04ca0081c82c126d66d3d9677d8cdb5a37)